docs/platform-concepts/agi-token.md

```diff
@@ -18,28 +18,28 @@ micro_nav: true
     
 ---
 
-SingularityNET uses the AGI Token for its marketplace. The AGI Token is an ERC-20 token hosted on the Ethereum Blockchain. Right now, we are in the `beta` phase of the SingularityNET Marketplace. That means that you will need to use Kovan or Ropsten Testnet AGI to work with our tools. You can get Kovan or Ropsten Testnet AGI by following the instructions [here](#agi-faucet).
+SingularityNET uses the AGIX Token for its marketplace. The AGIX Token is an ERC-20 token hosted on the Ethereum Blockchain. Right now, we are in the `beta` phase of the SingularityNET Marketplace. That means that you will need to use Kovan or Ropsten Testnet AGIX to work with our tools. You can get Kovan or Ropsten Testnet AGIX by following the instructions [here](#agix-faucet).
 
-## Testnet AGI vs Mainnet AGI
+## Testnet AGIX vs Mainnet AGIX
 For those new to Ethereum, it may be a bit confusing what exactly the difference between Testnet and Mainnet tokens are. In principle, you can distinguish them as follows:
 * Testnet tokens are used to test software applications with, try out demos, or in our case the Beta.
 * Mainnet tokens are used for officially deployed software.
 * Testnet tokens never have a monetary value.
 * Mainnet tokens could have a monetary value.
 * Both tokens can be used to add extra utility to the software.
 
-Since we are currently in the Beta stage, we only use Kovan and Ropsten Testnet AGI. You can find our Mainnet AGI Token (to be used in a later stage) [here](https://etherscan.io/address/0x8eb24319393716668d768dcec29356ae9cffe285) with the address `0x8eb24319393716668d768dcec29356ae9cffe285`.
+Since we are currently in the Beta stage, we only use Kovan and Ropsten Testnet AGIX. You can find our Mainnet AGIX Token (to be used in a later stage) [here](https://etherscan.io/address/0x8eb24319393716668d768dcec29356ae9cffe285) with the address `0x8eb24319393716668d768dcec29356ae9cffe285`.
```

Comment:
Change the address

Reply:
@anandrgitnirman  Please change the address